articles/native-platforms/ios-objc/01-login.md

```diff
@@ -187,36 +154,37 @@ A0TwitterAuthenticator *twitter = [A0TwitterAuthenticator newAuthenticatorWithKe
+Now you are ready to implement the Login using Lock. You only need to instantiate and present it from any of your `UIViewControllers`, like this:
 
 ${snippet(meta.snippets.use)}
 
-[![Lock.png](/media/articles/native-platforms/ios-objc/Lock-Widget-Screenshot.png)](https://auth0.com)
+![Lock.png](/media/articles/native-platforms/ios-objc/Lock-Widget-Screenshot.png)
 
-> **Note**: There are multiple ways of implementing the login box. What you see above is the Login Widget, but if you want, you can use [your own UI](/libraries/lock-ios/use-your-own-ui).
-> Or you can also try our passwordless Login Widgets: [SMS](/libraries/lock-ios#sms) or [TouchID](/libraries/lock-ios#touchid)
+::: panel-info Login UI Options
+There are multiple ways of implementing the login box. What you see above is the Login Widget. You can also [Build your own UI](/libraries/lock-ios/use-your-own-ui), or implement one of the passwordless Login Widgets: [SMS](/libraries/lock-ios#sms) or [TouchID](/libraries/lock-ios#touchid).
+:::
 
 On successful authentication, `onAuthenticationBlock` will yield the user's profile and tokens.
 
-> To learn how to save and manage the tokens and profile, please read [this guide](/libraries/lock-ios/save-and-refresh-jwt-tokens)
+**NOTE:** To learn how to save and manage the tokens and profile, see: [Lock iOS: Saving and Refreshing JWT Tokens](/libraries/lock-ios/save-and-refresh-jwt-tokens).
 
-### 5. Showing user information
+## 5. Showing user information
 
-After the user has logged in, we can use the `profile` object which has all the user information:
+After the user has logged in, you can use the `profile` object which contains all the user information:
 
 ```objc
   self.usernameLabel.text = profile.name;
   self.emailLabel.text = profile.email;
 ```
 
-> You can [click here](/user-profile) to find out all of the available properties from the user's profile or you can check [A0UserProfile](https://github.com/auth0/Lock.iOS-OSX/blob/master/Lock/Core/A0UserProfile.h). Please note that some of this depend on the social provider being used.
+## Additional Information
 
-### 6. We're done
+See [User Profile](/user-profile) to find out all of the available properties of the user profile. 
 
-You've implemented Login and Signup with Auth0 in iOS. You're awesome!.
+Also see: [A0UserProfile](https://github.com/auth0/Lock.iOS-OSX/blob/master/Lock/Core/A0UserProfile.h).
 
-> You can also <a href="/package/native-mobile-samples/master?path=iOS/profile-sample-swift&type=replace&filePath=iOS/profile-sample-swift/ProfileSample/Info.plist${account.clientParam}">download</a> our sample project that shows how to store/update your user profile with Auth0
+You can also download a [sample project](/package/native-mobile-samples/master?path=iOS/profile-sample-swift&file_path=iOS/profile-sample-swift/ProfileSample/Info.plist&type=replace&client_id=${account.clientId}) that shows how to store/update your user profile with Auth0.
```

Comment:
I did everything but fix this link. I don't understand the sample project structure or how to locate the correct link. Will update with correct link when found.